RclrTests/RclrTests.csproj

```diff
@@ -17,6 +17,7 @@
     <RepositoryUrl>https://github.com/jmp75/rclr</RepositoryUrl>
     <PackageProjectUrl>https://github.com/jmp75/rclr</PackageProjectUrl>
     <PackageReleaseNotes>Migration to .NET Standard 2.0 and supporting R 3.5.x</PackageReleaseNotes>
```

Comment:
outdated

Reply:
Yeah, there are tons of outdated stuff.. have to remove/update it step by step.